aldryn_client/client.py

```diff
@@ -513,16 +518,99 @@ def workspace_init_virtualenv(self, sitename, path):
         if not os.path.exists(dev_path):
             os.mkdir(dev_path)
 
-        import subprocess
-        subprocess.call(['virtualenv', virtualenv_path])
+        # fix requirements
+        with open(requirements_path, 'r') as f:
+            lines = f.readlines()
 
-        subprocess.call([pip_path, 'install', '-r', requirements_path])
+        excluded = ['mercurial', 'bzr', 'gyp', 'pygobject', 'pygpgme', 'cffi']
+
+        with open(requirements_path, 'w') as f:
+            for line in lines:
+                if not any(re.match(pat, line) for pat in excluded):
+                    f.write(line)
+
+        subprocess.call(['virtualenv', virtualenv_path])
+        subprocess.call([pip_path, 'install', 'pyOpenSSL==0.15.1', 'ndg-httpsclient==0.3.3', 'pyasn1==0.1.7', 'cryptography==0.8.2'])
```

Comment:
Those are actually fixed requirements coming from the website base project (https://github.com/aldryn/cloud-base-project/blob/master/Dockerfile#L18). I agree they should be put somewhere else, but even if we refactor it now it would have no effect until all containers would've been redeployed..